doc/source/io.rst

```diff
@@ -4093,6 +4093,27 @@ The key functions are:
 
 .. _io.bigquery_reader:
 
+Authentication
+''''''''''''''
+
+You will be authenticated to the specified BigQuery account via Google's Oauth2 mechanism.
+
+Primary auth method is as simple as following the
+prompts in a browser window which will be opened for you. Should the browser not
+be available, or fail to launch, a code will be provided to complete the process
+manually.  Additional information on the authentication mechanism can be found
```

Comment:
I haven't changed old functionality, so there's no need to update this part of doc